corehq/apps/data_cleaning/tables.py

```diff
@@ -0,0 +1,28 @@
+from django.utils.translation import gettext_lazy
+from django_tables2 import columns
+
+from corehq.apps.hqwebapp.tables.elasticsearch.records import CaseSearchElasticRecord
+from corehq.apps.hqwebapp.tables.elasticsearch.tables import ElasticTable
+from corehq.apps.hqwebapp.tables.htmx import BaseHtmxTable
+
+
+class CleanCaseTable(BaseHtmxTable, ElasticTable):
+    record_class = CaseSearchElasticRecord
+
+    class Meta(BaseHtmxTable.Meta):
```

Comment:
Why is this needed?

Reply:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/tables/htmx.py#L12-L15

without doing this the `Meta` object from the parent class isn't carried over. I tried digging into why for a bit, but the result is that this line is needed for django tables.

Reply:
(if it's not included the properties from `BaseHtmxTable`'s `Meta` object aren't carried over...based on local testing)

Reply:
thanks for explaining